.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 3.1
```

Comment:
Let's remove rubocop from RakeTasks and from dependencies

Reply:
fixed in 261d378